packages/frontend/src/hooks/useColumnTotals.ts

```diff
@@ -31,15 +31,14 @@ export const isSummable = (item: Item | undefined) => {
     if (isCustomDimension(item)) {
         return false;
     }
-    const numericTypes: string[] = [
-        DimensionType.NUMBER,
-        MetricType.NUMBER,
-        MetricType.COUNT,
-        MetricType.SUM,
-    ];
+    const numericTypes: string[] = [MetricType.COUNT, MetricType.SUM];
+    const isNumberDimension =
```

Comment:
Keep DimensionType.NUMBER support, which I believe is correct, and not related with this bugfix.

DimensionType.NUMBER === 'string' so we can't leave it in the numericType, as it matches `MetricType.NUMBER`